assets/sass/components/card.scss

```diff
@@ -130,7 +125,7 @@
 
 ::slotted(i){
   font-size: rem(24);
-  margin: -3px var(--icon-right) 0 0;
+  margin: -3px var(--card-icon-right) 0 0;
```

Comment:
This line may have actually fixed the issue I aimed to solve with https://github.com/iamproperty/iamproperty.github.io/pull/213 

Because I put the var value it changed on hover which I didn't notice until after... So I changed it to `rem(16)`